src/Forms/GridField/GridFieldEditButton.php

```diff
@@ -70,11 +70,7 @@ public function getUrl($gridField, $record, $columnName, $addState = true)
             'edit'
         );
 
-        if ($addState) {
-            $link = $this->getStateManager()->addStateToURL($gridField, $link);
-        }
-
-        return $link;
+        return $gridField->addOtherGridfieldStateToURL($link, $addState);
```

Comment:
I don't think `$addState` makes sense anymore - we probably _always_ want to add state. Deprecate the method parameter instead.

Reply:
DONE